README.md

```diff
@@ -51,14 +51,16 @@ FATA[0000] Post http:///var/run/docker.sock/v1.17/images/create?fromImage=cockro
 ```
 On OSX:
 ```bash
-# Setup Boot2Docker. This should only need to be done once.
-boot2docker init
-# Start Boot2Docker. This will need to be run once per reboot.
-boot2docker start
```

Comment:
The equivalent docker-machine command is `docker-machine start default`, we should still mention it.